src/effects/VST/VSTEffect.cpp

```diff
@@ -3815,10 +3863,22 @@ VSTEffectValidator::~VSTEffectValidator()
 }
 
 
-std::unique_ptr<EffectInstance::Message> VSTEffectValidator::MakeMessage(
-   VSTEffectSettings &settings)
+std::unique_ptr<EffectInstance::Message>
+VSTEffectWrapper::MakeMessageFS(VSTEffectSettings &settings) const
 {
-   return std::make_unique<VSTEffectMessage>(settings);
+   VSTEffectMessage::ParamVector paramVector;
+   paramVector.resize(settings.mParamsMap.size(), std::nullopt);
+   
+   ForEachParameter
```

Comment:
This `ForEachParameter` can remain as it is.  It's not needed to optimize this to make the play head movement smooth while dragging the sliders.

Reply:
ok